cloudbuild-k8s-generator.py

```diff
@@ -182,8 +182,9 @@ def main():
   args = parser.parse_args()
 
   # TODO(PR/346): Spark operator: "make app/verify" fails
-  # TODO(PR/516): SonarQube is flaky
-  skiplist = ['sonarqube', 'spark-operator']
+  # TODO(ISSUE): elastic-gke-logging: "make app/verify" fails
+  # TODO(ISSUE): elasticsearch: "make app/verify" fails
+  skiplist = ['elastic-gke-logging', 'elasticsearch', 'spark-operator']
```

Comment:
Elastic and elastic gke should not be skipped

Reply:
They should be fixed in https://github.com/GoogleCloudPlatform/click-to-deploy/pull/514.

Reply:
fixed